### ISSUE TITLE
Fail to install requirement text file issue

### ISSUE THREAD

**Issue by joozero** [ev2900/OpenSearch_kNN_Vector_Search#1]
Due to lack of cloud9's default storage volume(10GiB), installation failed with below message.
Collecting nvidia-nccl-cu12==2.18.1
  Downloading nvidia_nccl_cu12-2.18.1-py3-none-manylinux1_x86_64.whl (209.8 MB)
     |▏                               | 1.3 MB 43.9 MB/s eta 0:00:05ERROR: Could not install packages due to an OSError: [Errno 28] No space left on device

Increase this to at least 20GiB and reflect this to CloudFormation Stack would be recommended.

**Comment by ev2900** [comment]
Good catch. I'll look into fixing this issue.

**Comment by ev2900** [comment]
I added [resize_EBS.sh](https://github.com/ev2900/OpenSearch_kNN_Vector_Search/blob/main/resize_EBS.sh) to the repository. I also added instructions on how to run the script in the [README.md](https://github.com/ev2900/OpenSearch_kNN_Vector_Search/blob/main/README.md). 

This bash script will resize the EBS volume of the EC2 attached to the Cloud9 environment to 100 GB. 

Ideally I would to be able to deploy with a bigger volume via. the CloudFormation. However, CloudFormation doesn't support adjusting the EBS volume size of the EC2 instance attached to the Cloud9 environment. Given this limitation the bash script is the next best solution.

Thanks again for raising this issue.
